Package.swift

```diff
@@ -23,7 +23,16 @@ let package = Package(
         .testTarget(
             name: "BezierKitTests",
             dependencies: ["BezierKit"],
-            path: "BezierKit/BezierKitTests"
+            path: "BezierKit/BezierKitTests",
+            linkerSettings: [
+              // Extend stack size on WebAssembly since the default stack size of wasm-ld (64kb)
+              // is not enough for testing BezierKit.Utils.pairiteration, which heavily calls itself
+              // recursively.
+              .unsafeFlags(
+                    ["-Xlinker", "-z", "-Xlinker", "stack-size=\(248 * 1024)"],
+                    .when(platforms: [.wasi])
+              ),
```

Comment:
I guess this will work for the test target, but is there a way to ensure that anyone using BezierKit as a dependency will have enough stack space?

Reply:
Unfortunately, there is no stable way to check the current stack space has at least N bytes. Do you think it would be better to mention it in README?